src/datetime/serde.rs

```diff
@@ -24,6 +24,10 @@ pub struct MicroSecondsTimestampVisitor;
 #[derive(Debug)]
 pub struct MilliSecondsTimestampVisitor;
 
+#[doc(hidden)]
+#[derive(Debug)]
+pub struct Rfc2822Visitor;
```

Comment:
I believe you can make this private. The others are public for backwards compatibility.